fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_InitPage.js

```diff
@@ -371,10 +371,22 @@ class CDUInitPage {
         }
         mcdu.onRightInput[0] = async (value, scratchpadCallback) => {
             if (value === "") {
-                mcdu.setScratchpadText(
-                    (isFinite(getZfw()) ? (getZfw() / 1000).toFixed(1) : "") +
-                    "/" +
-                    (isFinite(getZfwcg()) ? getZfwcg().toFixed(1) : ""));
+                let zfw = undefined;
+                let zfwCg = undefined;
+                if (!!SimVar.GetSimVarValue("L:A32NX_BOARDING_STARTED_BY_USR", "bool")) {
```

Comment:
```suggestion
                if (!!SimVar.GetSimVarValue("L:A32NX_BOARDING_STARTED_BY_USR", "bool")) {
```

For QA: Add testing instruction to testers that the auto-populate will only grab the EFB's target ZFW values if boarding is in progress